resource_stress/app-memory-stress/memory-hog.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+## mandatory arg
+container_id=${CONTAINER_ID}
+
+## optional args
+memory_consumption=${MEMORY_CONSUMPTION:=500M}
+duration=${DURATION:=60}
+ramp_time=${RAMP_TIME:=5}
+
+if [ ! -z "${CONTAINER_ID}" ]; then 
+   
+	echo "wait for the specified ramp time of ${ramp_time}s before injecting chaos"
+	sleep ${ramp_time}
+
+        echo "starting memory consumption of ${memory_consumption} Megabytes"
+
+        docker exec ${container_id} sh -c "apt-get update && apt-get install stress-ng -y && stress-ng  --vm 1 --vm-bytes ${memory_consumption} -t ${duration}"
```

Comment:
This is an issue. It assumes target containers are debian based & have apt installed. In cpu hog experiment we avoided this via reuse of a common platform agnostic command `md5sum /dev/zero` which works on vast majority. 

We either need to identify a native approach like that or make use of stress containers that can run in SYS_ADMIN mode to manipulate the processes in namespace of target container

Reply:
okay karthik will come up with such approach soon Thanks

Reply:
done @ksatchit PTAL